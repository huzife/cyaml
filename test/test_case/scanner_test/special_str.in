---
empty: |
preserve: |
  this
  is
  a
  preserve
  string
fold: >-
  this is a
  fold string
