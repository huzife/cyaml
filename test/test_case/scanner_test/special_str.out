(STREAM_START, )#
(DOC_START, ---)#
(BLOCK_MAP_START, )#
(KEY, empty)#
(SCALAR, )#
(KEY, preserve)#
(SCALAR, this
is
a
preserve
string
)#
(KEY, fold)#
(SCALAR, this is a fold string)#
(BLOCK_MAP_END, )#
(STREAM_END, )#
